main/streams/transports.c

```diff
@@ -76,7 +76,7 @@ PHPAPI php_stream *_php_stream_xport_create(const char *name, size_t namelen, in
 	if (persistent_id) {
 		switch(php_stream_from_persistent_id(persistent_id, &stream)) {
 			case PHP_STREAM_PERSISTENT_SUCCESS:
-				/* use a 0 second timeout when checking if the socket
+				/* use a 0-second timeout when checking if the socket
```

Comment:
I am not a native English speaker, why this change?

Reply:
I didn't know the specific grammar/punctuation rule at play (native speakers don't usually study the grammar of their own language), but this did look more correct to my eyes. A Google search quickly revealed numerous web pages which explain the rule at play. See the subheading "Hyphen with compound modifiers: multiple-word adjectives before nouns" at https://www.grammarly.com/blog/hyphen/.